Formula/libgrape-lite.rb

```diff
@@ -0,0 +1,62 @@
+class LibgrapeLite < Formula
+  desc "C++ library for parallel graph processing"
+  homepage "https://github.com/alibaba/libgrape-lite"
+  url "https://github.com/alibaba/libgrape-lite/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "aafc8e5e9b122c4bde4370ee3f2ab8afb043f2a1547cacc062991ad420e3b9a4"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  depends_on "glog"
+  depends_on "open-mpi"
+
+  on_linux do
+    depends_on "gcc"
```

Comment:
Does GCC 5 not work?

Reply:
I was assuming `depends_on "gcc"` will bring the default gcc. I will delete this part.

Reply:
Fixed.